### PR TITLE
handbrakecli: add rip DVD examples; use long arguments

### PR DESCRIPTION
**Pull request by mtausig**
- [ ] ~~The page (if new), does not already exist in the repository.~~
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

1.3.1+ds1-1build1

### CODE REVIEW

File: pages/common/handbrakecli.md [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
-`handbrakecli --preset="Android" -i {{input.ext}} -o {{output.mp4}}`
+`handbrakecli --preset="Android" --input {{input.ext}} --output {{output.mp4}}`
+
+- Print the content of a DVD, getting the CSS keys in the process:
+
+`handbrakecli --input {{/dev/sr0}} --title 0`
+
+- Rip the first track of a DVD in the specified device. Audiotracks and subtitle languages are specified as lists:
+
+`handbrakecli -input {{/dev/sr0}} --title 1 --output {{out.mkv}} --format av_mkv --encoder x264 --subtitle {{1,4,5}} --audio {{1,2}} --aencoder copy --quality {{23}}`
```
Comment (sbrl) [inline pages/common/handbrakecli.md:28]: Typo?
```suggestion
`handbrakecli --input {{/dev/sr0}} --title 1 --output {{out.mkv}} --format av_mkv --encoder x264 --subtitle {{1,4,5}} --audio {{1,2}} --aencoder copy --quality {{23}}`
```

Comment (mtausig) [reply pages/common/handbrakecli.md]: Yes, typo. (Even though both version would have worked)

**Comment by mtausig** [comment]
Squashed.

**Review by navarroaxel — APPROVED** [review]

**Comment by sbrl** [comment]
Hey! Next time, perhaps you could avoid squashing please? It makes it harder to tell what's changed when reviewing, and we've got a special button that allows us to squash easily on merge :smiley_cat:

**Review by sbrl — APPROVED** [review]
Thanks, @mtausig!

**Pull request merged**